src/main/java/io/odpf/firehose/config/KafkaConsumerConfig.java

```diff
@@ -3,7 +3,7 @@
 /**
  * The interface for configurations required to instantiate a consumer.
  */
-public interface KafkaConsumerConfig extends AppConfig {
+public interface KafkaConsumerConfig extends AppConfig, FilterConfig {
```

Comment:
ahh. this is not correct. kafkaconfig should not extend FilterConfig

Reply:
fixed